src/pages/_app.tsx

```diff
@@ -31,6 +33,14 @@ function MyApp({ Component, pageProps }) {
         {page}
       </Layout>
     ));
+
+  const router = useRouter();
+  let canonicalUrl = 'https://docs.amplify.aws';
```

Comment:
This is hardcoded to the prod domain. I was thinking about maybe pulling this from an env variable so that we could pass down different domains like the staging ones. But I'm not sure if we would really need this often or not. Open to any feedback on it